system/mp/mpcomm.cpp

```diff
@@ -2976,8 +3030,6 @@ void stopMPServer()
         MPserver->stop();
         delete MPserver;
         MPserver = NULL;
-        ::Release(worldcomm);
```

Comment:
Addressed by separate PR (gh-7812).
However, not sure why I took these lines out, still better to clear at this point. I will reinstate (in PR for HPCC-14244)